codebasin.py

```diff
@@ -80,19 +80,19 @@ def guess_project_name(config_path):
                         help='dump out annotated platform/parsing tree to <file.json>')
     parser.add_argument('--batchmode', dest='batchmode', action='store_true', default=False,
                         help="Set batch mode (additional output for bulk operation.)")
-    args = parser.parse_args()
+    argv = parser.parse_args()
```

Comment:
I think this one should remain `args`:
```python
args = parser.parse_args()
```
... is the idiomatic way of writing this, and what `parse_args()` returns is not a list of argument strings.

(This comment applies to everything in codebasin.py.)

Reply:
Agreed, because even the `argparse` function is called `parse_args` and not `parse_argv` ;)